sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -95,6 +95,12 @@ object ScalaJSPluginInternal {
       "Files used to compute this value (can be used in FileFunctions later).",
       KeyRanks.Invisible)
 
+  val scalaJSInstallJsdom = TaskKey[File](
```

Comment:
To follow our conventions, `JSDOM` should be all-caps (acronyms in camelCase are all caps unless they're in the first position)